lib/plugins/leetcode.js

```diff
@@ -534,15 +534,10 @@ plugin.login = function(user, cb) {
 };
 
 function parseCookie(cookie, body, cb) {
-  const isCN = config.app === 'leetcode.cn';
   const SessionPattern = /LEETCODE_SESSION=(.+?)(;|$)/;
   let csrfPattern;
-  if (isCN) {
-    csrfPattern = /name="csrfmiddlewaretoken" value="(.*?)"/;
-  } else {
-    csrfPattern = /csrftoken=(.+?)(;|$)/;
-  }
-  const reCsrfResult = csrfPattern.exec(isCN? body: cookie);
+  csrfPattern = /csrftoken=(.+?)(;|$)/;
```

Comment:
const csrfPattern should be fine if we drop `if else`

Reply:
Thank you for your comment! I will fix it right now, can you also help me to take a look at #56 please?